### PR TITLE
Pdbxmmcif-gemmi

### PR DESCRIPTION
**Pull request by agrossfield**
---
name: pdbxmmcif
about: Add support for pdbxmmcif
title: 'Add support for pdbxmmcif'
labels: 'enhancement'
assignees: ''
---

## Changes proposed in this pull request
    - Add support for reading pdbx/mmcif files
    - New dependency on package gemmi

## Checklist
    - Does this pull request address an open issue? If so, which one?  No
    - Have you applied a formatter to your code (e.g. flake8 or black for python, clang-format for C++)?

Yes

### CODE REVIEW

**Comment by agrossfield** [comment]
@tromo : I'd appreciate it if you could look at this before I merge.

Not immediately relevant, but before we do the next release and conda-forge package, we'll need to add the gemmi dependency to the recipe

**Comment by agrossfield** [comment]
The build fail on OSX is because the gemmi package only contains the .a file and not the .so, and they used an older version of the compiler suite. During the discussion of the issue I raised before, they said they planned to turn on building the so by default and re-render the package. I just created a new issue (https://github.com/conda-forge/gemmi-feedstock/issues/65) in hopes they don't forget.

Basically, we can't/shouldn't merge until this is fixed on their end.

**Comment by agrossfield** [comment]
Note to @tromo : we need to specify gemmi version 0.6.4 or higher when we update the conda-forge package
